app/routes/authenticate.js

```diff
@@ -25,20 +27,67 @@ module.exports = {
   method: 'GET',
   path: '/authenticate',
   options: {
-    auth: false
+    auth: { mode: 'try' }
   },
   handler: async (request, h) => {
     try {
-      const user = randomise()
+      if (auth.authType === 'dev') {
```

Comment:
I know this isn't part of your revert, but do we need this anymore? As we have our own OneLogin test accounts, we probably bypass this anyway don't we?

Reply:
Agree we should remove.  Plan to do in a future PR though, as will require updated tests etc.